blockchain/v2/reactor.go

```diff
@@ -302,23 +300,25 @@ func (r *BlockchainReactor) demux() {
 	var lastRate = 0.0
 	var lastHundred = time.Now()
 
-	var (
```

Comment:
I personally prefer var. We should agree on something instead of changing back / forth

Reply:
I don’t mind the var, this was to put the defer statements after the variables they apply to. We can either have four var blocks or put all defers at the end.